### PR TITLE
[DISCL-258] Render PixelStream FPS statistics from Qml

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: src/core/PixelStream.cpp [hunk 1]
```diff
@@ -191,20 +207,34 @@ void PixelStream::adjustSegmentRendererCount(const size_t count)
...
         segmentRenderers_.clear();
         for (size_t i=0; i<count; ++i)
-            segmentRenderers_.push_back( PixelStreamSegmentRendererPtr(new PixelStreamSegmentRenderer(renderContext_)) );
+            segmentRenderers_.push_back( boost::make_shared<PixelStreamSegmentRenderer>( ));
     }
 }
 
+void PixelStream::refreshSegmentsList( const deflect::PixelStreamSegments& segments )
+{
+    qDeleteAll( segmentsList_ );
```
Comment (tribal-tec) [inline src/core/PixelStream.cpp]: that does operator delete? This happens every frame? new/delete every frame should be avoided if possible.

File: src/core/resources/style.js [hunk 2]
```diff
@@ -17,3 +17,9 @@ var buttonsHeight = 100;
 var buttonsSpacing = 10;
 var buttonsEnabledOpacity = 100;
 var buttonsDisabledOpacity = 20;
+
+var statisticsBorderMargin = 10
```
Comment (tribal-tec) [inline src/core/resources/style.js]: ; at the end to be a proper javascript file

**Comment by tribal-tec** [comment]
+1

File: src/core/PixelStream.cpp (hunk 1 above)
Comment (rdumusc) [inline src/core/PixelStream.cpp]: yes delete. Then I will resize the list instead and add an update() function to the Segement class, it should be a bit more efficient.

File: src/core/resources/style.js (hunk 2 above)
Comment (rdumusc) [inline src/core/resources/style.js]: Oh, missed that because of QML where we don't put them.

**Comment by rdumusc** [comment]
Updated after code review!

**Pull request merged**